sematic/resolvers/worker.py

```diff
@@ -15,6 +15,14 @@
 
 # Sematic
 import sematic.api_client as api_client
+
+# Note: this will not actually import pytorch,
+# and it will only perform the patch if (a) pytorch
+# is actually importable (b) we're running
+# in bazel (c) we're running in Linux
+# Needs to be done here in case user's func uses torch.
+# We want to make sure we can unpickle it.
```

Comment:
Shouldn't this import be placed in `sematic.bazel.worker` instead?

Reply:
no, because `sematic.bazel.worker` doesn't get used for pipelines where the `sematic_pipeline` bazel macro has `dev=True` set (in other words, our own dev pipelines). Yet we still want this import to occur there.